src/pybind/mgr/dashboard/frontend/src/app/shared/api/rgw-bucket.service.spec.ts

```diff
@@ -75,7 +75,7 @@ describe('RgwBucketService', () => {
     service.exists('foo').subscribe((resp) => {
       result = resp;
     });
-    const req = httpTesting.expectOne('api/rgw/bucket');
+    const req = httpTesting.expectOne('api/rgw/bucket/foo');
     expect(req.request.method).toBe('GET');
     req.flush(['foo', 'bar']);
```

Comment:
@nizamial09 Although not related to this PR, notice that the response we are returning from the bucket request is wrong as it's not the bucket data but a list of bucket names (which I think it's related to the previous behavior).
This should be fixed also in master.

Reply:
Agree. This should be fixed in master.